### DISCUSSION TITLE
[vipsthumbnail] Artifacts on generated thumbnail from mrxs file

### DISCUSSION THREAD

**Discussion by piciuok** [libvips/libvips#2353] [Q&A]
**Describe the bug**
Output file from **vipsthumbnail** has visible artifacts - they are placed randomly every generating of thumbnail.

**To Reproduce**
Run:
`vipsthumbnail -s 2000 80.mrxs --linear --smartcrop attention`
or
`vipsthumbnail -s 2000 80.mrxs `

**Expected behavior**
Fine image without artifact

**Screenshots**
![tn_80](https://user-images.githubusercontent.com/26445888/125473051-7efe6376-25d3-4fb7-bc86-427f937648c1.jpg)

**Environment**
 - OS: Linux Mint 20.1
 - Vips: 8.11

MRXS info: 
SLIDE_VERSION = 01.03
SLIDE_NAME = None
IMAGENUMBER_X = 312
IMAGENUMBER_Y = 780
SLIDE_CREATION_FINISHED = True
CURRENT_SLIDE_VERSION = 2.2
IMAGE_OVERLAP_MICROMETERS_X = 0
IMAGE_OVERLAP_MICROMETERS_Y = 0
SLIDE_POSITION_X = 0
SLIDE_POSITION_Y = 0
SLIDE_CONTENT = DIGITAL_SLIDE
SLIDE_CREATIONDATETIME = 14/10/2016 14:11:42
VIMSLIDE_CAMERA_REAL_BITDEPTH = 8
CONFOCAL = NO
DISK_POSITION = 0
FOCUS_MAP = AUTO
FOCUS_LIMIT = DISABLED
FLAT_FIELD_CORRECTION = DISABLED
EXTENDED_FOCUS_ALGORITHM = FL Image
VIMSLIDE_SLIDE_BITDEPTH = 8
CAMERA_TYPE = CIS VCC-FC60FR19CL
ADAPTER_SIZE = 1
OPTOVAR_SIZE = 1
OBJECTIVE_MAGNIFICATION = 40
OBJECTIVE_NAME = Plan-Apochromat
SLIDE_TYPE = SLIDE_TYPE_BRIGHTFIELD
CameraImageDivisionsPerSide = 4
VIMSLIDE_CP_BRIGHTNESS = 0
VIMSLIDE_CP_CONTRAST = 0
VIMSLIDE_CP_GAMMA = 1
VIMSLIDE_CP_COLOR_BALANCE_RED = 0
VIMSLIDE_CP_COLOR_BALANCE_GREEN = 0
VIMSLIDE_CP_COLOR_BALANCE_BLUE = 0
VIMSLIDE_SLIDE_ROTATION = 11.41409
VIMSLIDE_SLIDE_FLIPHORZ = False
VIMSLIDE_SLIDE_FLIPVERT = False

**Comment by jcupitt** [comment]
Hi @piciuok, this is a very unfortunate bug in gdkpixbuf. If you update to a more recent version, the holes will vanish.

**Comment by jcupitt** [comment]
Ooop, sorry, I mean pixman:

https://github.com/openslide/openslide/issues/291
